ci/docker/base.dockerfile

```diff
@@ -73,8 +67,11 @@ ENV CMPLR_ROOT=$CMPLR_ROOT \
     LIBRARY_PATH=$LIBRARY_PATH \
     LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 
-COPY ci/shared/scripts/setup-${SETUP_ID}.sh ${WF_TMP_DIR}/setup.sh
-RUN chmod +x ${WF_TMP_DIR}/setup.sh && . ${WF_TMP_DIR}/setup.sh
+# Prepare working directory
+ENV WF_TMP_DIR=/opt/workflows
 
-# Clean
-RUN rm -rf $WF_TMP_DIR
+# Run the setup scripts
+RUN --mount=type=bind,rw,source=ci,target=${WF_TMP_DIR} \
```

Comment:
Mounting as `rw` to allow saving customized setup script. Docker discards the written data anyways.

https://docs.docker.com/reference/dockerfile/#run---mounttypebind

Reply:
TODO: check if `--mount=type=cache` can be applied here.

https://docs.docker.com/reference/dockerfile/#run---mounttypecache